sources/packages/backend/apps/db-migrations/src/sql/Queue/Add-queue-configurations-values.sql

```diff
@@ -0,0 +1,291 @@
+-- Inserting values to 'queue_configurations' table. This will be part of table creation migration. So, no drop migration sql, this data will destroy with table in drop migration.
+INSERT INTO
+  queue_configurations(queue_name, queue_configuration)
+VALUES
+  (
+    'start-application-assessment',
+    '{ "attempts": 3, "backoff": 180000, "dashboardReadonly": false }' :: json
+  );
+
+INSERT INTO
+  queue_configurations(queue_name, queue_configuration)
+VALUES
+  (
+    'ier12-integration',
+    '{
+        "attempts": 3,
+        "backoff": 180000,
+        "dashboardReadonly": false,
+        "cron": "0 14 * * *"
+      }' :: json
+  );
+
+INSERT INTO
+  queue_configurations(queue_name, queue_configuration)
+VALUES
+  (
+    'cra-process-integration',
+    '{
+        "attempts": 3,
+        "backoff": 180000,
+        "dashboardReadonly": false,
+        "cron": "0 2,8,14,20 * * *"
+      }' :: json
+  );
+
+INSERT INTO
+  queue_configurations(queue_name, queue_configuration)
+VALUES
+  (
+    'cra-response-integration',
+    '{
+        "attempts": 3,
+        "backoff": 180000,
+        "dashboardReadonly": false,
+        "cron": "0 5,11,17,23 * * *"
+      }' :: json
+  );
+
+INSERT INTO
+  queue_configurations(queue_name, queue_configuration)
+VALUES
+  (
+    'sin-validation-process-integration',
+    '{
+        "attempts": 3,
+        "backoff": 180000,
+        "dashboardReadonly": false,
+        "cron": "0 17 * * *"
+      }' :: json
+  );
+
+INSERT INTO
+  queue_configurations(queue_name, queue_configuration)
+VALUES
+  (
+    'sin-validation-request-integration',
+    '{
+        "attempts": 3,
+        "backoff": 180000,
+        "dashboardReadonly": false,
+        "cron": "0 17 * * *"
+      }' :: json
+  );
+
+INSERT INTO
+  queue_configurations(queue_name, queue_configuration)
+VALUES
+  (
+    'pt-msfaa-process-integration',
```

Comment:
Even for workflows, we stopped using pt/ft and we started adding them as part-time/full-time.
I would prefer the extended version when possible, as a suggestion.